lib/trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0...9]
+  end
```

Comment:
Kindly note the difference between using two dots (..) and three dots (...)

- For example, in Ruby 1...9 gives you a range that doesn't include 9 
whereas 1..9 gives you a range that does include 9.

In your case trimming the name with three dots (...) does not include the 10th character on index 9. Thus the method returns an incorrect word that has 9 characters instead of 10.